lib/ansible/modules/cloud/amazon/aws_ses_identity_dkim.py

```diff
@@ -0,0 +1,172 @@
+    identity: example.com
+    state: disabled
+'''
+
+RETURN = '''
+dkim_attributes:
+    description: Dictionary with DKIM information
+    type: complex
+    returned: always
+    sample: {
+        "dkim_tokens": [],
+        "dkim_enabled": "True",
+        "dkim_verification_status": "Pending"
+    }
+    contains:
+        dkim_tokens:
+            descritpion: Array of the 3 DKIM verification tokens needed for DNS records
+            type: list
+            sample: ['u5d5fa5twxzdni4aaoffdqvgnvku5kvu', '6xzw7n3abimypkyan5s6kuse4nymrfo2', 'nkwigx63eervm4hyja2lwzhjcfqyhka7']
+        dkim_enabled:
+            description: Boolean indicating if DKIM email sending is enabled or disabled for identity
+            type: bool
+            sample: True
+        dkim_verification_status:
+            description: Status of the DKIM verification process (DNS verification)
+            type: str
+            sample: 'Pending'
+'''
+
+from ansible.module_utils.aws.core import AnsibleAWSModule
+from ansible.module_utils.ec2 import camel_dict_to_snake_dict, AWSRetry
+
+try:
+    from botocore.exceptions import BotoCoreError, ClientError
+except ImportError:
+    pass  # caught by imported HAS_BOTO3
+
+
+def get_identity_dkim_settings(module, client, identity):
+    try:
+        response = client.get_identity_dkim_attributes(Identities=[identity])
+    except (BotoCoreError, ClientError) as e:
+        module.fail_json_aws(e, msg='Failed to retrieve identity DKIM settings for {identity}'.format(identity=identity))
+    dkim_attributes = response['DkimAttributes']
+
+    return dkim_attributes[identity]
+
+
+def ses_verify_dkim_domain(module, client, identity):
+    domain = identity.split('@')[-1]
+    try:
+        response = client.verify_domain_dkim(Domain=domain)
+    except (BotoCoreError, ClientError) as e:
+        module.fail_json_aws(e, msg='Failed to start DKIM verification for {domain}.'.format(domain=domain))
+    dkim_tokens = response['DkimTokens']
+    return dkim_tokens
+
+
+def enable_identity_dkim_settings(module, client, identity):
+    try:
+        response = client.set_identity_dkim_enabled(DkimEnabled=True, Identity=identity)
+        return response
+    except (BotoCoreError) as e:
+        module.fail_json_aws(e, msg='Failed to enable DKIM for {identity}.'.format(identity=identity))
+    except (ClientError) as e:
+        pass
+        # The enable call will always return a ClientError, complaining about missing DNS record, but still changes setting to 'enabled'.
+        # So once DNS validation goes through it should work already, hence we ignore errors here.
+
+
+def disable_identity_dkim_settings(module, client, identity):
+    try:
+        response = client.set_identity_dkim_enabled(DkimEnabled=False, Identity=identity)
+    except (BotoCoreError, ClientError) as e:
+        module.fail_json_aws(e, msg='Failed to disable DKIM settings for {identity}'.format(identity=identity))
+    return response
```

Comment:
Stylistic, but since we're not doing anything sith the response here, prehaps just return directly on line 133 rather than assigning to a varible and returning after the catch